<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
  <class>CheatDialog</class>
  <widget class="QDialog" name="CheatDialog">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>862</width>
        <height>612</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="windowTitle">
      <string>Cheats</string>
    </property>
    <widget class="QLabel" name="labelTitle">
      <property name="geometry">
        <rect>
          <x>10</x>
          <y>10</y>
          <width>300</width>
          <height>31</height>
        </rect>
      </property>
      <property name="font">
        <font>
          <pointsize>10</pointsize>
        </font>
      </property>
      <property name="text">
        <string>Game Title - GameID</string>
      </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
        <rect>
          <x>10</x>
          <y>570</y>
          <width>841</width>
          <height>41</height>
        </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
          <spacer name="horizontalSpacer">
            <property name="orientation">
              <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
              <size>
                <width>40</width>
                <height>20</height>
              </size>
            </property>
          </spacer>
        </item>
        <item>
          <widget class="QPushButton" name="buttonNewCheat">
            <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
              </sizepolicy>
            </property>
            <property name="text">
              <string>Add Cheat</string>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="buttonDelete">
            <property name="text">
              <string>Delete</string>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="buttonSave">
            <property name="text">
              <string>Save</string>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="buttonClose">
            <property name="text">
              <string>Close</string>
            </property>
          </widget>
        </item>
      </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
        <rect>
          <x>10</x>
          <y>80</y>
          <width>551</width>
          <height>471</height>
        </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
        <item>
          <widget class="QTableWidget" name="tableCheats">
            <property name="showGrid">
              <bool>false</bool>
            </property>
            <property name="columnCount">
              <number>3</number>
            </property>
            <attribute name="horizontalHeaderVisible">
              <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
              <bool>false</bool>
            </attribute>
            <column>
              <property name="text">
                <string />
              </property>
            </column>
            <column>
              <property name="text">
                <string>Name</string>
              </property>
            </column>
            <column>
              <property name="text">
                <string>Type</string>
              </property>
            </column>
          </widget>
        </item>
      </layout>
    </widget>
    <widget class="QLabel" name="labelAvailableCheats">
      <property name="geometry">
        <rect>
          <x>10</x>
          <y>60</y>
          <width>121</width>
          <height>16</height>
        </rect>
      </property>
      <property name="text">
        <string>Available Cheats:</string>
      </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
        <rect>
          <x>580</x>
          <y>440</y>
          <width>271</width>
          <height>111</height>
        </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
          <widget class="QPlainTextEdit" name="textNotes" />
        </item>
      </layout>
    </widget>
    <widget class="QLabel" name="labelNotes">
      <property name="geometry">
        <rect>
          <x>580</x>
          <y>420</y>
          <width>111</width>
          <height>16</height>
        </rect>
      </property>
      <property name="text">
        <string>Notes:</string>
      </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
        <rect>
          <x>580</x>
          <y>80</y>
          <width>271</width>
          <height>311</height>
        </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
          <widget class="QPlainTextEdit" name="textDetails" />
        </item>
      </layout>
    </widget>
    <widget class="QLabel" name="labelDetails">
      <property name="geometry">
        <rect>
          <x>580</x>
          <y>60</y>
          <width>55</width>
          <height>16</height>
        </rect>
      </property>
      <property name="text">
        <string>Details:</string>
      </property>
    </widget>
  </widget>
  <resources />
  <connections />
</ui>
